src/cmd/dev.go

```diff
@@ -50,10 +52,61 @@ func NewDevCommand() *cobra.Command {
 	cmd.AddCommand(NewDevFindImagesCommand(v))
 	cmd.AddCommand(NewDevGenerateConfigCommand())
 	cmd.AddCommand(NewDevLintCommand(v))
+	cmd.AddCommand(NewDevInspectCommand(v))
 
 	return cmd
 }
 
+// NewDevInspectCommand creates the `dev inspect` sub-command.
+func NewDevInspectCommand(v *viper.Viper) *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "inspect",
+		Short: "Commands to get information about a Zarf package using a `zarf.yaml`",
+	}
+
+	cmd.AddCommand(NewDevInspectDefinitionCommand(v))
+	return cmd
+}
+
+// DevInspectDefinitionOptions holds the command-line options for 'dev inspect definition' sub-command.
+type DevInspectDefinitionOptions struct {
+	Flavor       string
+	SetVariables map[string]string
+}
+
+// NewDevInspectDefinitionCommand creates the `dev inspect definition` sub-command.
+func NewDevInspectDefinitionCommand(v *viper.Viper) *cobra.Command {
+	o := &DevInspectDefinitionOptions{}
+
+	cmd := &cobra.Command{
+		Use:   "definition [ DIRECTORY ]",
+		Args:  cobra.MaximumNArgs(1),
+		Short: "Displays the fully rendered package definition",
+		Long:  "Displays the 'zarf.yaml' definition of a Zarf after package templating, flavors, and component imports are applied",
+		RunE:  o.Run,
+	}
+
+	cmd.Flags().StringVarP(&o.Flavor, "flavor", "f", "", lang.CmdPackageCreateFlagFlavor)
+	cmd.Flags().StringToStringVar(&o.SetVariables, "set", v.GetStringMapString(common.VPkgCreateSet), lang.CmdPackageCreateFlagSet)
+
+	return cmd
+}
+
+// Run performs the execution of 'dev inspect definition' sub-command.
+func (o *DevInspectDefinitionOptions) Run(cmd *cobra.Command, args []string) error {
+	ctx := cmd.Context()
+	pkg, err := layout2.LoadPackage(ctx, setBaseDirectory(args), o.Flavor, o.SetVariables)
+	if err != nil {
+		return err
+	}
+	pkg.Build = v1alpha1.ZarfBuildData{}
```

Comment:
Is this required to be set to an empty struct?

Reply:
Technically no, but most of the data is missing at this point. IMO since the package isn't built yet, it makes sense to not include the build data

Reply:
Well would the build data not be empty already here?

Reply:
No migrations and I believe one other field get values in LoadPackage